changelog.mdx

```diff
@@ -5,6 +5,43 @@ metaDescription: Chalk's latest updates.
 description: Updates to Chalk!
 ---
 
+## December 23, 2024
+
+### Search and filter logs by query name, log message, deployment, and more!
+
+You can now search logs in the Logs Explorer in the dashboard using a key-value pair query format:
+`key1: value1 key2: value2 ...`. This allows you to combine multiple filters at once. The Logs Explorer
+displays the available keys for filtering, shown below.
+
+![Updated log explorer](/img/log-search-filters.png)
+
+### Search by file name in the source code viewer
+
+In the Source Code viewer, found in the Deployments page of the dashboard, you can now search for files by name
+in addition to the existing text search through the source code from the linked deployment.
+
+![Source code viewer with file search](/img/view-source-file-explorer.png)
+
+### Include query context in metadata field
+
+The Chalk Client now offers an [include_metadata](/api-docs#ChalkClient.query.include_meta) option to associate
```

Comment:
This is wrong. It's ChalkClient.query.query_context, and it's for moving arbitrary key-value pairs into a resolver

Reply:
i need to write docs, will do today